shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/repositories/OnboardingRepository.kt

```diff
@@ -0,0 +1,60 @@
+package com.mbta.tid.mbta_app.repositories
+
+import androidx.datastore.core.DataStore
+import androidx.datastore.preferences.core.Preferences
+import androidx.datastore.preferences.core.edit
+import androidx.datastore.preferences.core.stringSetPreferencesKey
+import co.touchlab.skie.configuration.annotations.DefaultArgumentInterop
+import com.mbta.tid.mbta_app.model.OnboardingScreen
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.flow.map
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+
+interface IOnboardingRepository {
+    suspend fun getPendingOnboarding(): List<OnboardingScreen>
+
+    suspend fun markOnboardingCompleted(screen: OnboardingScreen)
+}
+
+class OnboardingRepository : IOnboardingRepository, KoinComponent {
+    private val accessibilityStatus: IAccessibilityStatusRepository by inject()
+    private val dataStore: DataStore<Preferences> by inject()
```

Comment:
question(non-blocking): Why consume from the dataStore directly instead of from SettingsRepo? It seems ideal for maintaining clean boundaries between repos to limit the number of places that read from the data store directly.